### PR TITLE
[Detail] Issue id: #1763 add security_monitoring_system demo Solution: when megnetic sensor is triggered,buzzer will make sound and camera will caputure image and upload to the iot platform.we can monitor the system on mobile app.

### PR DESCRIPTION
**Pull request by joxhehe**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 1bb44732e3383e221f17fdc2c34b246e08007c41

**Review by ethanlcz — APPROVED** [review]
looks good to me.

**Comment by skylarCai** [comment]
auto build success,number:188

**Comment by skylarCai** [comment]
autotest fail,number:1495

File: haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/code/main.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+import display      # 显示库
+import network      # 网络库
+import ucamera      # 摄像头库
+import _thread      # 线程库
+from cloudAI import *
+
+# 物联网平台连接标志位
+iot_connected = False
+wlan = None
+# 三元组信息
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/code/main.py:27]: LGFM

File: haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/code/main.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+productKey = "Your-ProductKey"
+deviceName  = "Your-devicename"
+deviceSecret  = "Your-deviceSecret"
+# 物联网设备实例
+device = None
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/code/main.py:33]: LGFM

File: haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/README.md [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+> Python脚本的详细说明请参考脚本内嵌的文字注释
+
+
+1. **修改路由器名称及密码**
+
+&emsp;&emsp;
+修改security_monitoring_system工程里main.py中SSID和PWD的值为开发者实际要连接的路由器的名称及密码（请注意名称和密码都需要放在''符号中间）。
+
+```python
+# Wi-Fi SSID和Password设置
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/README.md:234]: LGFM

File: haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/README.md [hunk 4]
```diff
@@ -0,0 +1,321 @@
...
+```
+
+&emsp;&emsp;
+修改完成之后connect_wifi函数就会连接开发者自己设定的路由器。
+
+2. **修改设备的三元组信息**
+
+&emsp;&emsp;
+按照创建产品、设备步骤获取三元组信息，填入main.py中：
+```python
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/README.md:246]: LGFM

File: haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/README.md [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+
+&emsp;&emsp;
+推送此脚本到M5StackCore2之后，通过PC端串口调试工具可以看到设备端打印如下日志。日志的含义为:
+* “link platform connected” 代表成功连接到云端平台。
+* "upload--->"之后跟着的为设备端向云端发送的请求信息，其中"eventName"为"saveImageTest"，表示调用云端增值服务中的获取图片url信息功能。
+* "download <----"之后跟着的是收到云端回复的识别结果信息：
+    - result：url是否获取成功。（success：成功，failed：失败）
+    - imageUrl：返回的图片url链接地址
+    - commandName：saveImageTestReply表明该消息是saveImageTest服务的返回信息。
+```json
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/security_monitoring_system/m5stack/README.md:264]: @ethanlcz please check

**Pull request merged**